features/other_a/invite_queue.feature

```diff
@@ -35,6 +35,13 @@ Feature: Invite queue management
     When I follow "Get an Invitation"
     Then I should see "Request an Invitation"
 
+  Scenario: An admin can delete people from the queue
+    Given an invitation request for "invitee@example.org"
+      And I am logged in as an admin
+      And I go to the manage invite queue page
```

Comment:
Just a tiny nitpick here! Cucumbers usually have Given steps for setting up background, When steps for performing actions, and Then steps for describing results (https://cucumber.io/docs/reference#steps). It ultimately doesn't change how anything works, but for style, I'd suggest making this `When I go to the manage invite queue page` instead.